misc/release-docker.sh

```diff
@@ -13,12 +13,12 @@ fi
 
 docker pull docker.pkg.github.com/v6d-io/v6d/vineyardd:$version
 docker pull docker.pkg.github.com/v6d-io/v6d/vineyardd:latest
-docker pull docker.pkg.github.com/v6d-io/v6d/vineyard-controller:nightly
+docker pull docker.pkg.github.com/v6d-io/v6d/vineyard-operator:nightly
 
 docker tag docker.pkg.github.com/v6d-io/v6d/vineyardd:$version vineyardcloudnative/vineyardd:$version
 docker tag docker.pkg.github.com/v6d-io/v6d/vineyardd:latest vineyardcloudnative/vineyardd:latest
-docker tag docker.pkg.github.com/v6d-io/v6d/vineyard-controller:nightly vineyardcloudnative/vineyard-controller:nightly
+docker tag docker.pkg.github.com/v6d-io/v6d/vineyard-operator:nightly vineyardcloudnative/vineyard-operator:nightly
```

Comment:
Seems that we don't have a `docker.pkg.github.com/v6d-io/v6d/vineyard-operator:nightly`, @dashanji could you please double check the image tag here?

We use `nightly` as the nightly build (generated by each CI runs) and `:latest` as the latest released version.

Reply:
Make sense, I'll check it now.

Reply:
/cc @sighingnow. I have added it in the new pr https://github.com/v6d-io/v6d/pull/1002.